cypress/e2e/accessibility.cy.ts

```diff
@@ -21,6 +21,9 @@ describe("Accessibility tests in light mode", () => {
         cy.visit("/calendar");
         cy.get(".light-button").click();
 
+        // Wait for calendar to finish loading
```

Comment:
I don't think these comments are necessary - I think the code is great without!

Reply:
Done - comments removed